server/item/recipe/register.go

```diff
@@ -0,0 +1,14 @@
+package recipe
+
+// recipes is a list of each recipe.
+var recipes []Recipe
+
+// Register registers a new recipe.
+func Register(recipe Recipe) {
+	recipes = append(recipes, recipe)
+}
+
+// Recipes returns each recipe in a slice.
+func Recipes() []Recipe {
+	return recipes
```

Comment:
Would maybe copy the recipe slice here so that people cannot directly modify the slice and edit the recipes with it.